docs/documentation/cn/sql-reference/sql-statements/Data Definition/create-function.md

```diff
@@ -39,6 +39,10 @@ CREATE [AGGREGATE] FUNCTION function_name
 >           "finalize_fn": 聚合函数获取最后结果的函数签名。对于聚合函数是可选项，如果没有指定，将会使用默认的获取结果函数
 >
 >           "md5": 函数动态链接库的MD5值，用于校验下载的内容是否正确。此选项是可选项
+> 
+>           "prepareFnSymbol": 自定义函数的prepare函数的函数签名，用于从动态库里面找到prepare函数入口。此选项对于自定义函数是可选项
```

Comment:
keep consistent with other flag.

rename it prepare_fn

Reply:
done